torchrl/envs/transforms/transforms.py

```diff
@@ -4094,41 +4093,30 @@ def __init__(
 
     def reset(self, tensordict: TensorDictBase) -> TensorDictBase:
         """Resets episode rewards."""
-        # Non-batched environments
-        _reset = tensordict.get("_reset", None)
-        if _reset is None:
-            _reset = torch.ones(
-                self.parent.done_spec.shape if self.parent else tensordict.batch_size,
-                dtype=torch.bool,
-                device=tensordict.device,
-            )
+        _reset = tensordict.get("_reset", None)  # Try to get reset from root
```

Comment:
How is this supposed to work when you have one reward for agent1 and one for agent2, and their reset differ?
Seems a bit hard to do a generic logic.
Shouldn't we have as many reward transforms as agents, each pointing to a specific done signal?